modules/local/itsx_cutasv.nf

```diff
@@ -0,0 +1,31 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options    = initOptions(params.options)
+
+process ITSX_CUTASV {
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    conda (params.enable_conda ? "bioconda::itsx=1.1.3" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/itsx:1.1.3--hdfd78af_1"
+    } else {
+        container "quay.io/biocontainers/itsx:1.1.3--hdfd78af_1"
+    }
+
+    input:
+    path(fasta)
+    
+    output:
+    path(  "ASV_ITS_seqs.full.fasta" ), emit: fasta
+    //path( "*addSpecies.fna*"), emit: addspecies
+
+    script:
```

Comment:
Please output the software version, e.g. as in 
https://github.com/nf-core/ampliseq/blob/adcae8a113d3c0b4c636edcd242614dd846b6d13/modules/local/dada2_addspecies.nf#L58
https://github.com/nf-core/ampliseq/blob/adcae8a113d3c0b4c636edcd242614dd846b6d13/modules/local/dada2_addspecies.nf#L32
https://github.com/nf-core/ampliseq/blob/adcae8a113d3c0b4c636edcd242614dd846b6d13/modules/local/dada2_addspecies.nf#L28

and than forward into software summary via e.g. https://github.com/nf-core/ampliseq/blob/adcae8a113d3c0b4c636edcd242614dd846b6d13/workflows/ampliseq.nf#L290

Reply:
That was easily done with your detailed instructions, thank you!